### PR TITLE
interfaces/mount: keep track of kept mount entries

### PR DESCRIPTION
**Pull request by zyga**
This assists in computing the effective current profile as all
the kept and mounted things are in in.

Signed-off-by: Zygmunt Krynicki <zygmunt.krynicki@canonical.com>

### CODE REVIEW

File: interfaces/mount/change.go [hunk 1]
```diff
@@ -31,6 +31,8 @@ import (
 type Action string
 
 const (
+	// Keep indicates that a given mount entry should be kept as-is.
+	Keep Action = "keep"
```
Comment (morphis) [inline interfaces/mount/change.go]: What actually uses this keyword?

Comment (zyga) [inline interfaces/mount/change.go]: snap-update-ns does (in upcoming branch)

Comment (morphis) [inline interfaces/mount/change.go]: Ah ok, that is why I didn't found it anywhere.

**Comment by morphis** [comment]
LGTM

**Pull request merged**